src/game/Maps/SpawnManager.cpp

```diff
@@ -160,6 +158,9 @@ void SpawnManager::RespawnAll()
 
 void SpawnManager::Update()
 {
+    std::move(m_spawnInserts.begin(), m_spawnInserts.end(), std::back_inserter(m_spawns));
+    m_spawnInserts.clear();
+    std::sort(m_spawns.begin(), m_spawns.end());
```

Comment:
Yeah this solution wont float. This is going to be extremely expensive once we port everything to spawn groups.

Reply:
dynguid*

Reply:
it's apparently been causing other issues too, but I can't see a solution that doesn't do this *and* preserves `AddCreature()` in LoadFromDB